doc/clustering.md

```diff
@@ -0,0 +1,213 @@
+X-Etcd-Cluster-Id: 2d51087373879c4a
+
+{
+    "members": [
+        {
+            "clientURLs": [
+                "http://192.168.199.105:32379"
+            ], 
+            "id": "53d7e5cda976b1c", 
+            "name": "pd3", 
+            "peerURLs": [
+                "http://192.168.199.105:32380"
+            ]
+        }, 
+        {
+            "clientURLs": [
+                "http://192.168.199.105:12379"
+            ], 
+            "id": "4c97c22075384b7c", 
+            "name": "pd1", 
+            "peerURLs": [
+                "http://192.168.199.105:12380"
+            ]
+        }, 
+        {
+            "clientURLs": [
+                "http://192.168.199.105:22379"
+            ], 
+            "id": "ce80f2badea42715", 
+            "name": "pd2", 
+            "peerURLs": [
+                "http://192.168.199.105:22380"
+            ]
+        }
+    ]
+}
+```
+
+## A 3 nodes local cluster with docker
+
+```bash
+
+# Set correct HostIP here. 
+export HostIP="192.168.199.105"
+
+# Start pd1
+docker run -d -p 11234:1234 -p 19090:9090 -p 12379:2379 -p 12380:2380 --name pd1 \
+        pingcap/pd  \
+        --addr="0.0.0.0:1234" --advertise-addr=${HostIP}:11234 \
+        --http-addr="0.0.0.0:9090" \
+        --cluster-id=1 \
+        --etcd-name=pd1 \
+        --etcd-advertise-client-url="http://${HostIP}:12379" \
+        --etcd-advertise-peer-url="http://${HostIP}:12380" \
+        --etcd-initial-cluster="pd1=http://${HostIP}:12380,pd2=http://${HostIP}:22380,pd3=http://${HostIP}:32380" \
+        --etcd-listen-peer-url="http://0.0.0.0:2380" \
+        --etcd-listen-client-url="http://0.0.0.0:2379" 
+       
+# Start pd2
+docker run -d -p 21234:1234 -p 29090:9090 -p 22379:2379 -p 22380:2380 --name pd2 \
+        pingcap/pd  \
+        --addr="0.0.0.0:1234" --advertise-addr=${HostIP}:21234 \
+        --http-addr="0.0.0.0:9090" \
+        --cluster-id=1 \
+        --etcd-name=pd2 \
+        --etcd-advertise-client-url="http://${HostIP}:22379" \
+        --etcd-advertise-peer-url="http://${HostIP}:22380" \
+        --etcd-initial-cluster="pd1=http://${HostIP}:12380,pd2=http://${HostIP}:22380,pd3=http://${HostIP}:32380" \
+        --etcd-listen-peer-url="http://0.0.0.0:2380" \
+        --etcd-listen-client-url="http://0.0.0.0:2379" 
+        
+# Start pd3
+docker run -d -p 31234:1234 -p 39090:9090 -p 32379:2379 -p 32380:2380 --name pd3 \
+        pingcap/pd  \
+        --addr="0.0.0.0:1234" --advertise-addr=${HostIP}:31234 \
+        --http-addr="0.0.0.0:9090" \
+        --cluster-id=1 \
+        --etcd-name=pd3 \
+        --etcd-advertise-client-url="http://${HostIP}:32379" \
+        --etcd-advertise-peer-url="http://${HostIP}:32380" \
+        --etcd-initial-cluster="pd1=http://${HostIP}:12380,pd2=http://${HostIP}:22380,pd3=http://${HostIP}:32380" \
+        --etcd-listen-peer-url="http://0.0.0.0:2380" \
+        --etcd-listen-client-url="http://0.0.0.0:2379" 
+```
+
+Using `docker ps` to see cluster starts ok:
```

Comment:
+Using `docker ps` to see the running containers and check if the cluster is started: